### ISSUE TITLE
Reason to freeze_batchnorm 

### ISSUE THREAD

**Issue by marijnl** [KevinMusgrave/powerful-benchmarker#38]
I sampled some results from the benchmark result page and see you tend to freeze_batchnorm=True. What is the reason to freeze_batchnorm?

**Comment by KevinMusgrave** [comment]
Some people found that freezing the batch norm parameters helps avoid overfitting, especially on smaller datasets. This is assuming that you're starting with a pretrained network.
